README.md

```diff
@@ -185,6 +185,14 @@ Alternatively, to run a "smaller" set of tests, from the main repo folder run `p
 
 Note that some figures in the original paper were generated using the Python code at https://github.com/motiwari/BanditPAM-python. That code is not pretty, nor is it maintained. It only exists for reference and for reproducibility of the plots.
 
+## Documentation
+
+The documents of BanditPAM can be found here: s/documents/documentation
```

Comment:
The shorthand `s/documents/documentation` means "change `documents` to `documentation`", like in the syntax for `sed`